zeppelin-plugins/launcher/yarn/src/main/java/org/apache/zeppelin/interpreter/launcher/YarnRemoteInterpreterProcess.java

```diff
@@ -242,13 +244,15 @@ private ContainerLaunchContext setUpAMLaunchContext() throws IOException {
 
     // Set the resources to localize
     this.stagingDir = new Path(fs.getHomeDirectory() + "/.zeppelinStaging", appId.toString());
+    LOGGER.info("Use staging directory: {}", this.stagingDir);
     Map<String, LocalResource> localResources = new HashMap<>();
 
     File interpreterZip = createInterpreterZip();
     Path srcPath = localFs.makeQualified(new Path(interpreterZip.toURI()));
     Path destPath = copyFileToRemote(stagingDir, srcPath, (short) 1);
     addResource(fs, destPath, localResources, LocalResourceType.ARCHIVE, "zeppelin");
-    FileUtils.forceDelete(interpreterZip);
+    LOGGER.info("Add zeppelin archive: {}", destPath);
+    //FileUtils.forceDelete(interpreterZip);
```

Comment:
Does it mean that commenting without erasing is a temporary modification?

Reply:
Right, I have uncommented it now.